.ci/install_cc_env_ubuntu.sh

```diff
@@ -29,42 +30,19 @@ chronic sudo apt-get install -y rpm2cpio
 echo "Update apt repositories"
 chronic sudo apt-get update
 
-echo "Install linux-container kernel"
-chronic sudo apt-get install -y --force-yes linux-container
-
 echo "Install qemu-lite binary"
 chronic sudo apt-get install -y --force-yes qemu-lite
 
-echo "Download clear containers image"
-latest_version=$(curl -sL https://download.clearlinux.org/latest)
-curl -LO "https://download.clearlinux.org/current/clear-${latest_version}-containers.img.xz"
-
-echo "Validate clear containers image checksum"
-curl -LO "https://download.clearlinux.org/current/clear-${latest_version}-containers.img.xz-SHA512SUMS"
-sha512sum -c clear-${latest_version}-containers.img.xz-SHA512SUMS
-
-echo "Extract clear containers image"
-unxz clear-${latest_version}-containers.img.xz
-
+clear_release=$(curl -sL https://download.clearlinux.org/latest)
 cc_img_path="/usr/share/clear-containers"
-cc_img_link_name="clear-containers.img"
-sudo mkdir -p ${cc_img_path}
-echo "Install clear containers image"
-sudo install --owner root --group root --mode 0755 clear-${latest_version}-containers.img ${cc_img_path}
 
-echo -e "Create symbolic link ${cc_img_path}/${cc_img_link_name}"
-sudo ln -fs ${cc_img_path}/clear-${latest_version}-containers.img ${cc_img_path}/${cc_img_link_name}
+"./$cidir/install_clear_image.sh" ${clear_release} "${cc_img_path}"
 
 bug_url="https://github.com/clearcontainers/runtime/issues/91"
+kernel_clear_release=12760
```

Comment:
This kernel is version is fixed because we newer ones oops on semaphore CI (issue https://github.com/clearcontainers/runtime/issues/91). We should at some point use a versions.txt (we could have different configurations, one versions.txt for semaphore CI, one for devs). That part is captured in https://github.com/clearcontainers/tests/issues/24.